tracing/tracer.go

```diff
@@ -0,0 +1,103 @@
+package tracing
+
+import (
+	"fmt"
+
+	"github.com/opentracing/opentracing-go"
+	zipkinot "github.com/openzipkin-contrib/zipkin-go-opentracing"
+	zipkin "github.com/openzipkin/zipkin-go"
+	"github.com/openzipkin/zipkin-go/reporter"
+	zipkinhttp "github.com/openzipkin/zipkin-go/reporter/http"
+	"github.com/reddit/baseplate.go/log"
```

Comment:
nit: Please add a blank line before this line, so that we have 3 import groups: stdlibs, third party packages, and packages from the same repo/go module.